### ISSUE TITLE
QR code link is broken

### ISSUE THREAD

**Issue by Jean85** [joindin/joindin-web2#924]
The QR code link in the talk page appears to be broken.

I.e, here: https://joind.in/event/phpday-2024/symfony-messenger-the-sharpest-tool-in-your-php-toolbox

The `QR-Code (opens in new window)` appears to me like this in the browser: [QR-Code (opens in new window)](https://chart.googleapis.com/chart?cht=qr&chs=300x300&chl=https%3A%2F%2Fjoind.in%2Ftalk%2F92ac5%3Fqr&choe=UTF-8&chld=H)

**Comment by heiglandreas** [comment]
Yeah. Google discontinued QR-code generation via chart.googleapis.com. Instead one should use the nerw charts.google.com. But a search for "QR" does not result in anything....

**Comment by Jean85** [comment]
It seems that the point is that they discontinued the image generation endpoint: https://developers.google.com/chart/image

**Comment by heiglandreas** [comment]
Yeah... Please use https://www.qr-code-generator.com/ )or some other generator) for now until we found something decent as replacement...

Sorry for the inconvenience!

**Comment by Jean85** [comment]
Is https://packagist.org/packages/bacon/bacon-qr-code a possible replacement?

**Comment by heiglandreas** [comment]
Could be. But a service that can be used via a get request would definitely be prefered 😁 

Yes! We do not need to handle that. And Yes, we are doomed when someone decides to no longer support that 🙈

**Comment by Jean85** [comment]
Roger. As for a temporary workaround, https://forqrcode.com/ is far better, the one you suggested tries to steer you toward signup trial and then forces its short-linking onto you :(

**Comment by iansltx** [comment]
...and work stuff hit like a sledgehammer. I can maybe take a look at this _this_ weekend, but won't be able to touch it prior to phpDay, though if a PR comes up I can review it.

**Comment by DannyvdSluijs** [comment]
As pointed out by Max Fomichev in the DPC Slack (#speakers-2025 private channel)
> Max Fomichev  Today at 15:03
looks like this google's chart api is deprecated, but you can always generate your own, the link in the talk itself.
Also you can get the 404 link and use [quickchart.io](http://quickchart.io/) instead of https://chart.googleapis.com/
so instead of:
https://chart.googleapis.com/chart?cht=qr&chs=300x300&chl=https%3A%2F%2Fjoind.in%2Ftalk%2Fdfc37%3Fqr&choe=UTF-8&chld=H
do:
https://quickchart.io/chart?cht=qr&chs=300x300&chl=https%3A%2F%2Fjoind.in%2Ftalk%2Fdfc37%3Fqr&choe=UTF-8&chld=H

@heiglandreas this seems to satisfy you need of a service that offers a GET request. I've created #927 for this.
